### ISSUE TITLE
Rename enterprise howto to rdbms (filename)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10657]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#979|
|Type|enhancement|
|Created|2005-04-13 21:45:41Z|
|Branch|https://github.com/twisted/twisted/tree/enterprise-rdbms-979|

Attachments:

* [enterprise-rdbms-979.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7dc/7dcd5ff96b80f1bb263084ecb01e7696484613fd/90039f02934e04d0cf759ac3bf624d54a19e5b91.patch) (7520 bytes) - added by thijs on 2008-07-13 11:06:25Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__979 979
type__enhancement enhancement
reporter__itamarst itamarst
priority__normal normal
milestone__ 
branch__branches_enterprise_rdbms_979 branches/enterprise-rdbms-979
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1113428741000000 1113428741000000
changetime__1217881986000000 1217881986000000
version__ 
owner__ 
cc__itamarst cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
probably help readers a bit.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement set owner to edsuom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

See attached [[enterprise-rdbms-979.patch](None/enterprise-rdbms-979.patch) patch].

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

a branch would be better, since the patch changes some markup and deletes the entire enterprise howto. ;)  Combinator docs linked from http://divmod.org/trac/wiki/DivmodCombinator

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [24445]) Branching to 'enterprise-rdbms-979'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra removed owner|
|-|-|

Replying to [exarkun](#note_5):
> a branch would be better, since the patch changes some markup and deletes the entire enterprise howto. ;)

Added the branch, tried to test it with the doc builder but got this error:

```
Server Connection Lost

Connection to distributed server lost:

[Failure instance: Traceback from remote host -- Traceback unavailable]
```

Up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

That failure means a build was pending.  It's a buildbot bug.  If you wait a while it eventually goes away and turns into real results:

http://buildbot.twistedmatrix.com/builders/documentation/builds/716

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra removed owner|
|-|-|

Found the issue and updated the branch, [tested](http://buildbot.twistedmatrix.com/builders/documentation/builds/721) it and putting it up for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_10"></a>|@therve set owner to @thijstriemstra|
|-|-|

There are still references of the file in row.xhtml and glossary.xhtml. Please fix those and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_11"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [therve](#note_10):
> There are still references of the file in row.xhtml and glossary.xhtml. Please fix those and merge.

I fixed the references to enterprise.xhtml in r24519, will merge now.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24521]) Merge enterprise-rdbms-979: Rename enterprise.xhtml howto to rdbms.xhtml

Author: thijs
Reviewer: exarkun, therve
Fixes: [#10657](https://github.com/twisted/twisted/issues/10657)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
